webview/src/experiments/components/table/Table.tsx

```diff
@@ -43,6 +50,7 @@ export const ExperimentGroup: React.FC<RowProp & InstanceProp> = ({
         instance={instance}
         contextMenuDisabled={contextMenuDisabled}
```

Comment:
Seems like we are still suppressing the full context menu whenever an experiment is running. Perhaps we need to now do this on an action-by-action basis as there is nothing to stop a user from "starring" multiple experiments on the fly whilst an experiment is running.